lib/Driver/ToolChains.h

```diff
@@ -45,10 +45,14 @@ class LLVM_LIBRARY_VISIBILITY GenericUnix : public ToolChain {
 
   virtual std::string getDefaultLinker() const;
 
+  virtual std::string getTarget() const;
```

Comment:
Sure thing! Here's a pull request for the already committed methods: https://github.com/apple/swift/pull/2134